scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3852,448 @@ def rvs(self, dim, size=1, random_state=None):
 
 
 unitary_group = unitary_group_gen()
+
+_mhg_doc_default_callparams = """\
+m : array_like
+    The number of each type of object in the population.
+    That is, :math:`m[i]` is the number of objects of
+    type :math:`i`.
+n : array_like
+    The number of samples taken from the population.
+"""
+
+_mhg_doc_callparams_note = \
+"""`m` must be an array of positive integers. Quantiles `x` must
+sum up to the sample size `n`. If the quantile :math:`i` contains
+values out of the range :math:`[0, m_i]` where :math:`m_i` is the
+number of objects of type :math:`i` in the population, appropriate
+values (like zero by ``pmf`` method) are returned by the called
+methods. If `m` or `n` contain negative values, the result will
```

Comment:
Negative `m` and `n` are out of the domain while we assume that the quantiles `x` are continuous. So, whenever `m` or `n` are out of their domain, it makes sense to put `nan` there irrespective of the quantiles. (similarly, it makes sense to fill in the zeros when `m` and `n` are in their domain and the `x` values are non-integral or negative)

Reply:
I see. There might be arguments for raising a `ValueError` instead of returning `nan`, but I think you should leave it alone because it's consistent with `multinomial` and it makes sense to return `nan` for some elements and valid values for others.